frontend/rolecall/src/app/performance/performance-editor.component.html

```diff
@@ -190,24 +212,36 @@ <h3 class="input-title location-title">Country</h3>
               <h3 class="input-title location-title">Venue</h3>
             </div>
             <div class="location-card app-card">
-              <textarea type="text" class="location-input" rows="1" [value]="state.step_1.venue"
-                (input)="onStep1Input('venue', $event)"></textarea>
+              <textarea type="text"
+                  class="location-input"
+                  rows="1"
+                  [value]="state.step_1.venue"
+                  (input)="onStep1Input('venue', $event)"></textarea>
             </div>
           </div>
           <div class="description-container">
             <h3 class="input-title">Description</h3>
             <div class="description-card app-card">
-              <textarea type="text" class="description-input" rows="2" [value]="state.step_1.description"
-                (input)="onStep1Input('description', $event)"></textarea>
+              <textarea type="text"
+                  class="description-input"
+                  rows="2"
+                  [value]="state.step_1.description"
+                  (input)="onStep1Input('description', $event)"></textarea>
             </div>
             <div class="segment-list app-card">
-              <div class="segment-container" *ngFor="let segment of state.step_3.segments">
-                <h3 class="title">{{segment['name']}}</h3>
-                <div class="pos-container" *ngFor="let pos of segment.custom_groups">
+              <div *ngFor="let segment of state.step_3.segments; let i = index"
+                  class="segment-container"
+                  [ngStyle]="{'margin-left': step2Data[i].siblingId > 0 ? '4rem' : '1rem' }">
+                  <h3 class="title">{{segment['name']}}</h3>
+                <div *ngFor="let pos of segment.custom_groups"
+                    class="pos-container"
+                    [ngStyle]="{'margin-left': hasRevelation && step2Data[i].siblingId < 1 ? '10rem' : '7rem' }">
```

Comment:
Will `siblingId` ever be negative? If not, maybe better to check for `siblingId === 0` instead of `siblingId < 1`

Reply:
I wrote it like that to save space(!). However, with the trick below I am saving so much space that I can easily afford === 0 . (BTW, the laugh above was supposed to go here.)